README.md

```diff
@@ -2,23 +2,24 @@
 
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-FSRS Helper is an Anki add-on that supports [FSRS4Anki](https://github.com/open-spaced-repetition/fsrs4anki) scheduler. It has six main features:
+FSRS Helper is an Anki add-on that supports [FSRS](https://github.com/open-spaced-repetition/fsrs4anki) algorithm. It has six main features:
 
 - **Reschedule** cards based on their entire review histories.
 - **Postpone** a selected number of due cards.
 - **Advance** a selected number of undue cards.
 - **Balance** the load during rescheduling (based on fuzz).
-- **Less Anki** on Easy Days (such as weekends) during rescheduling (based on load balance).
+- Less Anki on **Easy Days** (such as weekends) during rescheduling (based on load balance).
 - **Disperse** Siblings (cards with the same note) to avoid interference & reminder.
+- **Flatten** future due cards to a selected number of reviews per day.
 
 # Requirements
 
-- For Anki version in 2.1.55 - 2.1.66 
+- For Anki version in 2.1.55 - 2.1.66 (maintenance is stopped)
   - Enable V3 Scheduler
-  - FSRS4Anki scheduler version >= 3.0.0
+  - FSRS4Anki version >= 3.0.0
 - For Anki version >= 23.10
```

Comment:
Move this above the 2.1.55-2.1.66 part. I believe that the latest version should always be at the top, especially if the older version is no longer maintained.